### PR TITLE
Add timestamps for logs

### PR DESCRIPTION
**Pull request by SiarheiFedartsou**
# Issue

https://github.com/Project-OSRM/osrm-backend/issues/6374

## Tasklist

 - [ ] CHANGELOG.md entry ([How to write a changelog entry](http://keepachangelog.com/en/1.0.0/#how))
 - [ ] update relevant [Wiki pages](https://github.com/Project-OSRM/osrm-backend/wiki)
 - [ ] add tests (see [testing documentation](https://github.com/Project-OSRM/osrm-backend/blob/master/docs/testing.md))
 - [ ] review
 - [ ] adjust for comments
 - [ ] cherry pick to release branch

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -455,6 +455,11 @@ add_subdirectory(${FLATBUFFERS_SRC_DIR}
         ${CMAKE_CURRENT_BINARY_DIR}/flatbuffers-build
         EXCLUDE_FROM_ALL)
 
+set(FMT_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/third_party/fmt-9.1.0/include")
```
Comment (SiarheiFedartsou) [inline CMakeLists.txt:458]: Btw fmt promises to have super fast algorithm for floating point numbers rendering https://github.com/fmtlib/dtoa-benchmark#results

We can try to re-use it in our JSON renderer here https://github.com/Project-OSRM/osrm-backend/blob/ef8f3d7508dbdcaeac990d1b891d2e0775c594e6/include/util/json_renderer.hpp#L41

**Comment by mjjbell** [comment]
Is there a benefit to vendoring `fmt` vs adding as a Conan dependency?

**Comment by SiarheiFedartsou** [comment]
> Is there a benefit to vendoring `fmt` vs adding as a Conan dependency?

Well, we have to support ENABLE_CONAN=OFF use case and it is not clear how to deal with it.

**Comment by mjjbell** [comment]
Ah, I thought we had started using it all the time for some reason.

**Review by mjjbell — APPROVED** [review]

**Comment by SiarheiFedartsou** [comment]
> Ah, I thought we had started using it all the time for some reason.

Btw I would discuss that, tbh it is not clear what can be benefits of using dependencies from the system rather than "pinned" versions from Conan. I can just imagine that system dependencies may be of newer versions containing bug fixes etc. At the same time dependency management with Conan is much easier.

**Comment by nilsnolde** [comment]
> tbh it is not clear what can be benefits of using dependencies from the system

My 2c: I think it's mostly about not duplicating libraries (esp linked ones). Also package maintainers (brew, apt, pacman etc) really don't like runtime linked dependencies to be packaged with a library but use the stuff that's inside the package repos and sometimes go to great lengths to patch sources to accomplish that. The biggest issue though is probably linked shared libraries when you distribute OSRM as a package.. OSRM will fail to find the runtime (shared lib) dependencies if they're not installed system-wide (and/or if conan sets some RPATH in the exes/libs). And then it'd have to be compatible with whatever the system packages provides. But I'd guess that's less of a problem for software like OSRM/Valhalla, I personally would go with using conan by default, it's so much less painful for the maintainers to pin versions. And this is FOSS, if people need compat with older versions of deps they can open PRs:)

**Pull request merged**